sdk/core/core/inc/az_config.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_CONFIG_H
+#define _az_CONFIG_H
+
+#include <az_span.h>
+
+#include <_az_cfg_prefix.h>
+
+/*
+ *
+ */
+enum {
+  AZ_HTTP_REQUEST_URL_BUF_SIZE = 2* 1024,
+  AZ_HTTP_REQUEST_BODY_BUF_SIZE = 1024,
+  AZ_HTTP_REQUEST_HEADER_COUNT = 10,
+  AZ_HTTP_REQUEST_HEADER_BUF_SIZE = AZ_HTTP_REQUEST_HEADER_COUNT * sizeof(az_pair),
```

Comment:
I wouldn't put this here as having in this file implies that it is changeable independently of AZ_HTTP_REQUEST_HEADER_COUNT and this is forbidden. I think this should just be used where needed within the appropriate .c files(s). Or, the C files want to have this enum, then that's OK but each .c file's choice.

Reply:
To be clear you are suggesting that AZ_HTTP_HEADER_BUF_SIZE is not a constant but rather configured every its used as:
AZ_HTTP_REQUEST_HEADER_COUNT * sizeof(az_pair)